### DISCUSSION TITLE
Diagnose Direktanschluss an Steuerung

### DISCUSSION THREAD

**Discussion by mathias-ch1903** [openv/openv#524] [General]
Hallo zusammen

Das Viessmann Modul welches die Sende / Empfangsdioden beinhaltet scheint ja modular ausgeführt zu sein (siehe Viessmann Artikel No 7819323)

Hat schon jemand direkt an die Schnittstelle einen Adapter gebaut, ohne dabei über die Sende / Empfangsdioden zu gehen?
Wäre ja auch in der Steuerung steckbar möglich, wenn ich die Bilder vom Modul ansehe.

**Comment by Linde2022** [comment]
Hallo an Alle,

das würde mich auch sehr interessieren.
Ich möchte gern ein umfassendes Datelogging mit einem Windows-Rechner machen, da unsere Anlage nicht ganz fehlerfrei läuft. Die Einträge in den Meldungshistorien ergeben aber kein klares Bild.
Nun habe ich so gar keine Ahnung vom Löten und dergleichen.
Hat Jemand etwas für mich, das die Vitotronic 200 W01C mit meinem Rechner verbindet?

Danke schon mal.

**Reply by qwert4712** [reply]
wenn du schon ein Kabel hast, kommst du hier mit Vitosoft am schnellstem zu einem Ergebnis
https://connectivity.viessmann.com/de-de/mp_rt/vitosoft.html

**Comment by mathias-ch1903** [comment]
Hallo 

Die Vitotronic 200 KW2 hier habe ich inzwischen mittels eines CH340 USB Adapters (den ich aus Zeitgründen bestellt habe, Mailadresse siehe [1]) angebunden, über die Opto Schnittstelle [2], nicht direkt. Läuft 1A, auch mit 10m USB Verlängerung.

[1] https://github.com/openv/openv/wiki/Bauanleitung-USB
[2] https://github.com/openv/openv/wiki/Die-Optolink-Schnittstelle

**Comment by Linde2022** [comment]
Hallo und vielen Dank für die Tipps !
Es mangelt immer an der Zeit, gleich alles auszuprobieren. Ich hab ein 
USB-Kabel, dass an der optischen Schnittstelle der Vitotronic angesteckt 
wird. Am amderen Ende ist ein ganz normaler USB-Stecker. Der steckt 
normaler Weise im Vitoconnect-Modul. Was da an Daten an die App heraus 
gegeben wird ist aber absolut lächerlich. Jede "billige" chinesische 
Solaranlagen-Steuerung gibt ihrem Betreiber mehr Informationen.

Ich werde die Demo-Version von Vitosoft installieren und mal schauen, 
was sie mir an Betriebsdaten zeigt.
Vielen Dank und viele Grüße
Klaus

------ Originalnachricht ------
Von: "qwert089" ***@***.***>
An: "openv/openv" ***@***.***>
Cc: "Linde2022" ***@***.***>; "Comment" 
***@***.***>
Gesendet: 31.01.2022 20:25:49
Betreff: Re: [openv/openv] Diagnose Direktanschluss an Steuerung 
(Discussion #524)

>
>wenn du schon ein Kabel hast, kommst du hier mit Vitosoft am 
>schnellstem zu einem Ergebnis
>https://connectivity.viessmann.com/de-de/mp_rt/vitosoft.html
>
>—
>Reply to this email directly, view it on GitHub 
><https://github.com/openv/openv/discussions/524#discussioncomment-2084109>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AXN4GBKBDLJAKO3N2ZRTF3TUY3O33ANCNFSM5H3REVPA>.
>Triage notifications on the go with GitHub Mobile for iOS 
><https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
>or Android 
><https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>You are receiving this because you commented.Message ID: 
***@***.***>
>

**Reply by qwert4712** [reply]
Mit dem Tool kannst du sämtliche Kodierungen auslesen und als PDF Speichern --> Abnahmeprotokoll 
und aber auch die Datenpunkte die du möchtest grafisch darstellen oder als csv speichern.
Das USB Kabel muss dazu natürlich an den PC.

Meist muss man für das neue USB to Serial Device noch einen Treiber über Windows Update laden.
